packages/framework-integration-tests/src/config/config.ts

```diff
@@ -15,6 +15,7 @@ if (process.env.BOOSTER_ENV === 'local') {
 Booster.configure('development', (config: BoosterConfig): void => {
   config.appName = 'my-store'
   config.provider = AWS.Provider()
+  config.assets = ['assets', 'assetFile.txt']
```

Comment:
The user configures an array of folders and files that will be zipped with the lambda code. Later they will be able to read it inside command and event handlers